exasol_transformers_extension/deployment/deployment_utils.py

```diff
@@ -5,21 +5,28 @@
 import ssl
 from pathlib import Path
 from contextlib import contextmanager
-from jinja2 import Environment, PackageLoader, select_autoescape
-from exasol_transformers_extension.deployment import constants
+from jinja2 import Environment, PackageLoader, select_autoescape, ChoiceLoader
+from exasol_transformers_extension.deployment import constants, work_with_spans_constants, work_without_spans_constants
 
 
 logger = logging.getLogger(__name__)
 
 
-DB_PASSWORD_ENVIRONMENT_VARIABLE = f"TE_DB_PASSWORD"
+DB_PASSWORD_ENVIRONMENT_VARIABLE = f"TE_DB_PASSWORD" #todo should these move to constants?
```

Comment:
could be , don't think that anything speaks against it

Reply:
i checked with @ahsimb  and we don't actually need these anymore, will delete them